testing/conftest.py

```diff
@@ -136,6 +137,24 @@ def testdir(testdir: Testdir) -> Testdir:
     return testdir
 
 
+@pytest.fixture
+def symlink_or_skip():
```

Comment:
Slightly improved version used in https://github.com/pytest-dev/pytest/pull/6733.